frontend/src/lib/utils.tsx

```diff
@@ -1327,3 +1332,52 @@ export function calculateDays(timeValue: number, timeUnit: TimeUnitType): number
     }
     return timeValue
 }
+
+export function processCohort(cohort: CohortType, isNewCohortFilterEnabled: boolean = false): CohortType {
+    return {
+        ...cohort,
+        ...(isNewCohortFilterEnabled
+            ? {
+                  /* Populate value_property with value and overwrite value with corresponding behavioral filter type */
+                  filters: {
+                      properties: {
```

Comment:
I don't know the intent behind these lines, and I see this PR just copy/pasted them from another place, but could it be that this is missing?

```suggestion
                      ...cohort.filters,
                      properties: {
```

Reply:
Good note, but I don't think this is necessary since cohort.filters will always only have one property key called properties.